game/games/sortit/box.gd

```diff
@@ -14,6 +14,21 @@ export(float) var number_scale = 0.8
 
 var number: int = 0
 var is_black: bool = false
+var force_sleep: bool = false
+
+
+func _process(_delta):
+	# Handle boxes falling through the ground (Should not happen)
+	if translation.y < -10:
+		push_warning("SortIt: Box dropped through the ground")
+		_on_despawn_timer_timeout()
+
+	if force_sleep:
+		sleeping = true
+		$CollisionShape.disabled = true
+	else:
+		sleeping = false
+		$CollisionShape.disabled = false
```

Comment:
How about simplifying this?

```gdscript
sleeping = force_sleep
$CollisionShape.disabled = force_sleep
```
Also might be a good idea to cache `$CollisionShape` in an `onready var`?

Reply:
I changed the force_sleep logic.